File: PHP/DPR.php
<!-- /*Saya Rakha Dhifiargo Hariadi
 NIM 2209489 mengerjakan soal 
 Latihan praktikum 1 dalam mata
 kuliah Struktur data
 untuk keberkahanNya maka saya tidak
 melakukan kecurangan seperti 
 yang telah dispesifikasikan. Aamiin.*/ -->
<?php
class DPR
{
    private $id; // untuk id anggota dpr
    private $nama; // untuk nama anggota dpr
    private $bidang; // untuk bidang anggota dpr
    private $partai; // untuk partai anggota dpr
    private $foto; // untuk foto anggota dpr

    public function __construct($id = '', $nama = '', $bidang = '', $partai = '', $foto = '') // konstruktor
    {
        $this->id = $id;
        $this->nama = $nama;
        $this->bidang = $bidang;
        $this->partai = $partai;
        $this->foto = $foto;
    }

    // setter untuk atribut anggota dpr
    public function setId($id)
    {
        $this->id = $id;
    }

    public function setNama($nama)
    {
        $this->nama = $nama;
    }

    public function setBidang($bidang)
    {
        $this->bidang = $bidang;
    }

    public function setPartai($partai)
    {
        $this->partai = $partai;
    }

    public function setFoto($foto)
    {
        $this->foto = $foto;
    }

    // getter untuk atribut anggota dpr
    public function getId()
    {
        return $this->id;
    }

    public function getNama()
    {
        return $this->nama;
    }

    public function getBidang()
    {
        return $this->bidang;
    }

    public function getPartai()
    {
        return $this->partai;
    }

    public function getFoto()
    {
        return $this->foto;
    }

    function __destruct()
    {
        
    }
}
?>

File: python/DPR.py
# /*Saya Rakha Dhifiargo Hariadi
#  NIM 2209489 mengerjakan soal 
#  Latihan praktikum 1 dalam mata
#  kuliah Struktur data
#  untuk keberkahanNya maka saya tidak
#  melakukan kecurangan seperti 
#  yang telah dispesifikasikan. Aamiin.*/ 
class DPR:

    __id = 0; # untuk id anggota dpr
    __nama = ""; # untuk nama anggota dpr
    __bidang = ""; # untuk bidang anggota dpr
    __partai = ""; # untuk partai anggota dpr

    def __init__(self, id = 0, nama = "", bidang = "", partai = ""): # constructor
        self.__id = id
        self.__nama = nama
        self.__bidang = bidang
        self.__partai = partai

    # setter untuk setiap atribut
    def setId(self, id):
        self.__id = id

    def setNama(self, nama):
        self.__nama = nama

    def setBidang(self, bidang):
        self.__bidang = bidang

    def setPartai(self, partai):
        self.__partai = partai

    # getter untuk setiap atribut
    def getId(self):
        return self.__id
    
    def getNama(self):
        return self.__nama
    
    def getBidang(self):
        return self.__bidang
    
    def getPartai(self):
        return self.__partai
